docs/reference/commandline/network_ls.md

```diff
@@ -38,8 +39,40 @@ NETWORK ID                                                         NAME
 c288470c46f6c8949c5f7e5099b5b7947b07eabe8d9a27d79a9cbf111adcbf47   host                host                
 7b369448dccbf865d397c8d2be0cda7cf7edc6b0945f77d2529912ae917a0185   bridge              bridge              
 95e74588f40db048e86320c6526440c504650a1ff3e9f7d60a497c4d2163e5bd   foo                 bridge    
+63d1ff1f77b07ca51070a8c227e962238358bd310bde1529cf62e6c307ade161   dev                 bridge
 ```
 
+`--filter` is used to get rid of items user doesn't care from results, for
```

Comment:
I used the "filtering" section of the `docker ps` documentation to make it more consistent, wdyt?

    ## Filtering

    The filtering flag (`-f` or `--filter`) format is a `key=value` pair. If there
    is more than one filter, then pass multiple flags (e.g. `--filter "foo=bar" --filter "bif=baz"`).
    Multiple filter flags are combined as an `OR` filter. For example, 
    `-f type=custom -f type=builtin` returns both `custom` and `builtin` networks.

    The currently supported filters are:

    * id (network's id)
    * name (network's name)
    * type (custom|builtin)


    #### Type

    The `type` filter supports two values; `builtin` displays predefined networks
    (`bridge`, `none`, `host`), whereas `custom` displays user defined networks.

    The following filter matches all user defined networks:

    ```bash
    $ docker network ls --filter type=custom
    NETWORK ID          NAME                DRIVER
    95e74588f40d        foo                 bridge
    63d1ff1f77b0        dev                 bridge
    ```

    By having this flag it allows for batch cleanup. For example, use this filter
    to delete all user defined networks:

    ```bash
    $ docker network rm `docker network ls --filter type=custom -q`
    ```

    A warning will be issued when trying to remove a network that has containers
    attached.

    #### Name

    The `name` filter matches on all or part of a network's name.

    The following filter matches all networks with a name matching the `foo` string.

    ```bash
    $ docker network ls --filter name=foo
    NETWORK ID          NAME                DRIVER
    95e74588f40d        foo                 bridge
    ```

    #### ID

    The `id` filter matches on all or part of a network's ID.

    The following filter matches all networks with a name containing the
    `06e7eef01700` string.

    ```bash
    $ docker network ls --filter id=63d1ff1f77b07ca51070a8c227e962238358bd310bde1529cf62e6c307ade161
    NETWORK ID          NAME                DRIVER
    63d1ff1f77b0        dev                 bridge
    ```

    You can also filter for a substring in a ID as this shows:

    ```bash
    $ docker ps --filter id=95e74588f40d
    NETWORK ID          NAME                DRIVER
    95e74588f40d        foo                 bridge

    $ docker ps --filter id=95e
    NETWORK ID          NAME                DRIVER
    95e74588f40d        foo                 bridge
    ```